odoo/tools/mail.py

```diff
@@ -185,6 +185,11 @@ def html_sanitize(src, silent=True, sanitize_tags=True, sanitize_attributes=Fals
 
     logger = logging.getLogger(__name__ + '.html_sanitize')
 
+    # Convert mso conditionals to mako tags, otherwise they are stripped by the
+    # cleaner. Restore them afterwards.
+    src = src.replace(u'<!--[if mso]>', u'<%mso_if%>')
+    src = src.replace(u'<![endif]-->', u'<%mso_endif%>')
```

Comment:
Anyway seems hackish. Can't you find a cleaner way to keep your mso conditional tags and implement it properly in the cleaner itself ?